test/io/ferguson/read-class-reorder.chpl

```diff
@@ -0,0 +1,29 @@
+/*class Base {
+}
+
+class Parent : Base {
+  var x: int;
+  var y: int;
+}
+*/
+
+class Child /*: Parent*/ {
```

Comment:
I view it as a way of documenting this tension.  If we get far enough down the road that neither you nor I remember this issue when someone encounters it, it would be nice to be able to check the STATUS file and see that the behavior was at least expected, as opposed to something we hadn't even thought about.